src/components/Icon/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ClassNames from 'classnames';
+import styles from '@/components/Icon/Icon.css';
+
+import IconSearch from '@/assets/images/icons/web/search-simple.svg';
+
+type Category = 'search';
+
+interface Props {
+  description?: string;
+  heigth?: string;
+  name?: Category;
```

Comment:
We shouldn't be passing the name of an icon, but the image path. The icons shouldn't live inside the Icon component, but outside. We don't know which icons we will end up adding, and it will be up to the application using the library to define those. This is how the signature of the component should look like:

```
<Icon icon={arrow_path}
```

Reply:
Alright, I'll fix it